packages/fluentui/react-northstar/src/themes/teams/components/Divider/dividerStyles.ts

```diff
@@ -33,7 +33,7 @@ const dividerStyles: ComponentSlotStylesPrepared<DividerProps, DividerVariables>
         fontWeight: variables.importantFontWeight,
       }),
       ...(vertical && { height: '100%' }),
-      ...(childrenExist(children) || content
+      ...(hasChildren || hasContent
```

Comment:
```suggestion
      ...(hasContent
```

I suggest to keep only `hasContent`